[2024-08-29T16:55:39.268+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T16:55:39.276+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:55:33.705488+00:00 [queued]>
[2024-08-29T16:55:39.280+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:55:33.705488+00:00 [queued]>
[2024-08-29T16:55:39.280+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-08-29T16:55:39.299+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): tsk_load_to_s3> on 2024-08-29 16:55:33.705488+00:00
[2024-08-29T16:55:39.303+0000] {standard_task_runner.py:72} INFO - Started process 14577 to run task
[2024-08-29T16:55:39.307+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'zillow_analytics_dag', 'tsk_load_to_s3', 'manual__2024-08-29T16:55:33.705488+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/zillow_analytics.py', '--cfg-path', '/tmp/tmpme6zbl11']
[2024-08-29T16:55:39.308+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask tsk_load_to_s3
[2024-08-29T16:55:39.340+0000] {task_command.py:467} INFO - Running <TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:55:33.705488+00:00 [running]> on host ip-172-31-11-44.eu-west-3.compute.internal
[2024-08-29T16:55:39.408+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='says@master.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='zillow_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_load_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-08-29T16:55:33.705488+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-29T16:55:33.705488+00:00'
[2024-08-29T16:55:39.409+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T16:55:39.423+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-29T16:55:39.424+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'aws s3 mv /home/ubuntu/response_data_29082024165536.json s3://endtoend-bucket001/']
[2024-08-29T16:55:39.432+0000] {subprocess.py:86} INFO - Output:
[2024-08-29T16:55:40.053+0000] {subprocess.py:93} INFO - Completed 55.5 KiB/55.5 KiB (300.9 KiB/s) with 1 file(s) remainingmove: ../../home/ubuntu/response_data_29082024165536.json to s3://endtoend-bucket001/response_data_29082024165536.json
[2024-08-29T16:55:40.120+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-29T16:55:40.139+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T16:55:40.139+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=zillow_analytics_dag, task_id=tsk_load_to_s3, run_id=manual__2024-08-29T16:55:33.705488+00:00, execution_date=20240829T165533, start_date=20240829T165539, end_date=20240829T165540
[2024-08-29T16:55:40.165+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-29T16:55:40.180+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-29T16:55:40.189+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
